static/js/components.js

```diff
@@ -47,9 +42,13 @@ exports.toggle = function (opts) {
     };
 
     function select_tab(idx) {
+        var elem = meta.$ind_tab.eq(idx);
+        if (elem.hasClass('disabled')) {
+            return;
+        }
+
```

Comment:
We also need to remove unsubscribed streams from rendering into `All streams` tabs. Cause if guest user tries access All stream tab(by removing `disabled` class) guest user is able to see all the list of streams.